config/clusters/uwhackweeks/common.values.yaml

```diff
@@ -53,6 +53,7 @@ basehub:
             - name: home
               mountPath: /home/jovyan/shared
               subPath: _shared
+      # User image repo: https://github.com/ICESAT-2HackWeek/jupyter-image-2020
```

Comment:
@scottyhq is this correct? I couldn't find any other user image for the hackweek though this one seems to be updated quite a while ago

Reply:
we actually have a monorepo going this time around that includes content and image in one place https://github.com/ICESAT-2HackWeek/website2022